src/main/java/Calculator.java

```diff
@@ -0,0 +1,40 @@
+public class Calculator {
+
+    public static final Integer FULL_SIZE = 2;
+
+    public Integer progress(String numbersAndOperators) {
+        String[] strings = getSplitString(numbersAndOperators);
+        Integer result = getNumber(strings[0]);
+        int arrayLength = strings.length;
+
+        for (int i = 1; i < arrayLength; i+=2) {
+            arrayIndexValidation(arrayLength, i);
+            Integer number = getNumber(strings[i + 1]);
+            String operator = strings[i];
+
+            result = Operator.of(operator).getResult(result , number);
+        }
+
+        return result;
+    }
+
+    private void arrayIndexValidation(int arraySize, int index) {
+        if (arraySize <= index + 1) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+    private String[] getSplitString(String numberAndOperator) {
+        if (numberAndOperator == null || numberAndOperator.length() < 5){
+            throw new IllegalArgumentException();
```

Comment:
```suggestion
            throw new IllegalArgumentException("올바르지 않은 값");
```
에러메세지를 추가해 주면 어떨까요? 😄

Reply:
각 IllegalArgumentException마다 메세지 추가했습니다. 감사합니다.